rhc-ose-ansible/roles/openshift-management/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+openshift_token:
+openshift_master_url: localhost
+openshift_login_insecure_flag: --insecure-skip-tls-verify=true
+openshift_login_insecure: False
```

Comment:
the `openshift_login_insecure_flag` is used during the login play as it is a parameter to the `oc login` command